applications/nrf5340_audio/child_image/hci_ipc.conf

```diff
@@ -30,6 +30,10 @@ CONFIG_BT_CTLR_ADV_DATA_LEN_MAX=251
 CONFIG_MPSL_TRIGGER_IPC_TASK_ON_RTC_START=y
 CONFIG_MPSL_TRIGGER_IPC_TASK_ON_RTC_START_CHANNEL=4
 
+# Needed for builds with nrf21540
+#CONFIG_BT_CTLR_TX_PWR_ANTENNA=20
```

Comment:
Maybe we can set these to 10 dBm

Reply:
It's only comments, but I can change to 10 and then add a comment about 20